fedmsg/crypto/x509.py

```diff
@@ -65,11 +65,13 @@ def sign(message, ssldir=None, certname=None, **config):
 
     message['crypto'] = 'x509'
 
-    certificate = M2Crypto.X509.load_cert(
-        "%s/%s.crt" % (ssldir, certname)).as_pem()
+    marker = "-----BEGIN CERTIFICATE-----"
+    with open("%s/%s.crt" % (ssldir, certname), 'r') as f:
+        content = marker + f.read().split(marker)[-1]
+        certificate = M2Crypto.X509.load_cert_string(content).as_pem()
+
     # Opening this file requires elevated privileges in stg/prod.
-    rsa_private = M2Crypto.RSA.load_key(
-        "%s/%s.key" % (ssldir, certname))
+    rsa_private = M2Crypto.RSA.load_key("%s/%s.key" % (ssldir, certname))
```

Comment:
I recommend making this change in a separate commit since it is just formatting and is unrelated to the issue at hand.